[2021-07-17 13:59:47,045] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.bash_task4 2021-07-17T13:59:38.534695+00:00 [queued]>
[2021-07-17 13:59:47,050] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.bash_task4 2021-07-17T13:59:38.534695+00:00 [queued]>
[2021-07-17 13:59:47,050] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-17 13:59:47,050] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-17 13:59:47,050] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-17 13:59:47,057] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): bash_task4> on 2021-07-17T13:59:38.534695+00:00
[2021-07-17 13:59:47,060] {standard_task_runner.py:52} INFO - Started process 8952 to run task
[2021-07-17 13:59:47,062] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'parallel_dag', 'bash_task4', '2021-07-17T13:59:38.534695+00:00', '--job-id', '837', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpoq61w1ju', '--error-file', '/tmp/tmp2nqcan5n']
[2021-07-17 13:59:47,063] {standard_task_runner.py:77} INFO - Job 837: Subtask bash_task4
[2021-07-17 13:59:47,088] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.bash_task4 2021-07-17T13:59:38.534695+00:00 [running]> on host 120364900fc4
[2021-07-17 13:59:47,118] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=bash_task4
AIRFLOW_CTX_EXECUTION_DATE=2021-07-17T13:59:38.534695+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-17T13:59:38.534695+00:00
[2021-07-17 13:59:47,118] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-17 13:59:47,118] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-07-17 13:59:47,124] {subprocess.py:75} INFO - Output:
[2021-07-17 13:59:50,127] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-17 13:59:50,170] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=bash_task4, execution_date=20210717T135938, start_date=20210717T135947, end_date=20210717T135950
[2021-07-17 13:59:50,193] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-17 13:59:50,200] {local_task_job.py:151} INFO - Task exited with return code 0
